crates/syntax/test_data/parser/ok/0072_pub_tuple_field.rs

```diff
@@ -0,0 +1 @@
+struct MyStruct(pub (u32, u32));
```

Comment:
Is this handwritten? The test outputs here are generated from inline comments in the parser sources(see the comments starting with `test some_random_name`)